src/ScriptEditor/NetscriptDefinitions.d.ts

```diff
@@ -3707,6 +3707,20 @@ interface Stanek {
 /**
  * Collection of all functions passed to scripts
  * @public
+ * @remarks
+ * <b>Basic ns1 usage example:</b>
+ * ```ts
+ * property.methodName;
```

Comment:
Maybe we should give examples of both methods that are part of a sub-property, and methods that are not - and use real examples to avoid confusion.

e.g.

<b>Basic ns1 usage example:</b>
```ts
// Basic ns functions can be used directly
getHackingLevel();
// Some related functions are gathered within a common namespace
stock.getSymbols()
```

<b>Basic ns2 usage example:</b>
```ts
export async function main(ns) {
  // Basic ns functions can be accessed on the ns object
  ns.getHackingLevel();
  // Some related functions are gathered under a sub-property of the ns object
  ns.stock.getSymbols()
}
```

Reply:
Yeah, this would be idea on the actual specific method pages and showing how to use said method but on the parent ns page thing my original woudl suffice. But, if the dev asks me to do more will

Reply:
Umm, i might make some changed based on your feedback. But, i didn't really want to be too decriptive on the main ns page here https://github.com/danielyxie/bitburner/blob/dev/markdown/bitburner.ns.md I wanted to save your version for pages like property pages https://github.com/danielyxie/bitburner/blob/dev/markdown/bitburner.bladeburner.md and method pages https://github.com/danielyxie/bitburner/blob/dev/markdown/bitburner.bladeburner.getactionautolevel.md

Reply:
I don't know what I should do if i do it my way its going to take ages if I do it your way and be more descriptive i might be able to get away with not adding example on the property pages or method pages

Reply:
I will make slight changes using some of your suggestions, I still don't know if i should specific actually function names and properties in the examples. Anyways will update this pull request and see what you all think of the changes.